coalib/results/SourceRange.py

```diff
@@ -120,6 +120,45 @@ def expand(self, file_contents):
                                        tr.end.line,
                                        tr.end.column)
 
+    @enforce_signature
+    def affected_source(self, file_dict: dict):
+        r"""
+        Make an iterator that tells which are the lines affected in the
```

Comment:
I would have loved to just merge this but you're not making an iterator here but returning a tuple and that's a vital difference